src/AzureStack/SubscriptionsAdmin/Subscriptions.Admin.Tests/Subscriptions.Tests.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Subscriptions.Admin.Tests</PackageId>
+    <Description>Subscriptions.Admin.Tests Class Library</Description>
+    <AssemblyName>Subscriptions.Admin.Tests</AssemblyName>
+    <VersionPrefix>1.0.0</VersionPrefix>    
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
```

Comment:
Is this intentional

Reply:
This is for our local testing, not sure how it got added here.  We set the environment when we run live tests.

Reply:
So I'm guessing this is required